auction-house/program/src/execute_sale/mod.rs

```diff
@@ -1349,8 +1349,7 @@ fn auctioneer_execute_sale_logic<'c, 'info>(
                     &token_account.key(),
                     &seller.key(),
                     &[],
-                )
-                    .unwrap(),
+                )?,
```

Comment:
These are unrelated changes to workspaces right?
Just double checking ... probably should have been in separate PR unless it was needed to please clippy or whatever

Reply:
I neurotically fix things like this when I find them, but yeah would have been better as a separate PR.